cmd/memberagent/main.go

```diff
@@ -85,21 +86,51 @@ func main() {
 
 	mcNamespace := fmt.Sprintf(utils.NamespaceNameFormat, mcName)
 
-	err := retry.OnError(retry.DefaultRetry, func(e error) bool {
-		return true
-	}, func() error {
-		// Stat returns file info. It will return
-		// an error if there is no file.
-		_, err := os.Stat(tokenFilePath)
-		return err
-	})
-	if err != nil {
-		klog.ErrorS(err, " cannot retrieve token file from the path %s", tokenFilePath)
-		os.Exit(1)
+	identityKeyFile := os.Getenv("IDENTITY_KEY")
+	identityCertFile := os.Getenv("IDENTITY_CERT")
+	cABundleFile := os.Getenv("CA_BUNDLE")
```

Comment:
```suggestion
	caBundleFile := os.Getenv("CA_BUNDLE")
```
Words in names that are initialisms or acronyms (e.g. "URL" or "NATO") have a consistent case. 
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Change the title and `ca*` suggested 

Thanks for the review ;-)